bin/deployment/deploy

```diff
@@ -40,7 +40,7 @@ gendb() {
         $SUDO rm -rf /var/lib/candlepin/activemq-artemis/*
     else
         MESSAGE="Updating Database"
-        CHANGELOG="changelog-update.xml"
+        CHANGELOG="changelog-create.xml"
```

Comment:
Let's keep the "updated" name, as it better aligns with the verbiage used/expected for this type of operation.

Reply:
Liqubase will not allow it. the changelog-create name is in the DB for determining if the set has been run:
id + author + filename is the key.